models/Element/Tag.php

```diff
@@ -160,6 +160,10 @@ public static function getElementsForTag(
         return $tag->getDao()->getElementsForTag($tag, $type, $subtypes, $classNames, $considerChildTags);
     }
 
+    public static function getByPath($path) {
```

Comment:
please rename it to `getByNamePath` since it works only with name path.

Reply:
Wanted to use the same name as for data objects, assets etc. There it is also named `getByPath()` and not `getByFilenamePath()` for assets and `getByKeyPath()` for data objects.
Please confirm that I really should use a different name here.

Reply:
Ok, let's keep the name `getByPath()` for consistency. At least, add doc stating that it works only with name path. thanks!

Reply:
Fixed.